docs/detections/alerts-view-details.asciidoc

```diff
@@ -26,7 +26,11 @@ The *Overview* tab contains these features:
 
 * *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.
 
-* *Reason statement*: Provides a description of what generated the alert and provides general alert details. You can use this to understand the alert's origin and determine if the alert is relevant to your investigation.
+* *Alert rendered reason statement*: Summarizes the alert and renders the statement in a human-readable format. High-level alert details are provided, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Alert details are interactive. Hover over them to access multiple actions.
+
+* *Event rendered reason statement*: Surfaces important alert details to immediately provide useful context for understanding the alert. The statement is also rendered in a human-readable format and alert details are interactive.
```

Comment:
Consider replacing ~Event rendered reason statement~ with "Event renderer", because the alert renderer (today) shares the same content as the `kibana.alert.reason` field, but the Event renderer is unrelated to the "reason" field.